$OpenBSD$

Remove hardcoded cc and -O3
Append port's CFLAGS so we get our CFLAGS too.

Index: etc/Makefile.flat.template2
--- etc/Makefile.flat.template2.orig
+++ etc/Makefile.flat.template2
@@ -1,6 +1,6 @@
 
-CC=gcc
-CFLAGS=-O3 -DSPEC_CPU2004
+CC?=gcc
+CFLAGS+=-DSPEC_CPU2004
 LDFLAGS=-lm
 
 eprover: $(PROJ)
